src/app/_components/NextAuth/SignIn.tsx

```diff
@@ -0,0 +1,27 @@
+"use client";
+
+import { useSession } from "next-auth/react";
+import { useRouter } from 'next/navigation';
+import { signIn } from 'next-auth/react';
```

Comment:
This can go with the import for the useSession, so:

import { signIn, useSession } from "next-auth/react";

Just a heads up that there are a mix of single quotes and double quotes being used currently. This will be fixed en-masse once I set up ESLint, but still good to keep an eye out for as an individual dev.